features/step_definitions/stepdefs.js

```diff
@@ -524,6 +524,50 @@ Given('we follow {string}', async function (name) {
     }
 });
 
+Given('we are followed by:', async function (actors) {
+    for (const { name, type } of actors.hashes()) {
+        // Create the actor
+        this.actors[name] = await createActor(name, { type });
+
+        // Create the follow activity
+        const actor = this.actors[name];
+        const object = this.actors.Us;
+        const activity = await createActivity('Follow', object, actor);
+
+        this.activities[name] = activity;
```

Comment:
Not a big deal, but this is a bit of a vague name - not sure what a better one would be, we can leave it for now I reckon, until we actually need to use them!

Reply:
Ah, I missed this, copy pasta 🍝

Have updated in [4459752](https://github.com/TryGhost/ActivityPub/pull/233/commits/4459752c342a44ac22f7ddc88f0c8d504bc5cef0) 👍